test/unit/vpc-instance-token-manager.test.js

```diff
@@ -39,7 +39,11 @@ const IAM_PROFILE_ID = 'some-id';
 const EXPIRATION_WINDOW = 10;
 
 describe('VPC Instance Token Manager', () => {
-  const sendRequestMock = jest.fn().mockImplementation(() => ({ result: { access_token: TOKEN } }));
+  const sendRequestMock = jest.fn(); // .mockImplementation(() => ({ result: { access_token: TOKEN } }));
```

Comment:
```suggestion
  const sendRequestMock = jest.fn();
```
I guess this comment can be removed, right?

Reply:
fixed